lib/tdf3/src/models/encryption-information.ts

```diff
@@ -101,12 +101,12 @@ export class SplitKey {
         typeof metadata === 'object'
           ? JSON.stringify(metadata)
           : typeof metadata === 'string'
-          ? metadata
-          : () => {
-              throw new ConfigurationError(
-                "KAO generation failure: metadata isn't a string or object"
-              );
-            }
+            ? metadata
+            : () => {
+                throw new ConfigurationError(
+                  "KAO generation failure: metadata isn't a string or object"
+                );
+              }
```

Comment:
whitespace

Reply:
`make format` passed.  hmm

Reply:
yeah, I upgraded to prettier 3.3 which apparently has different rules for ternaries